pkgs/applications/audio/ardour/ardour6.nix

```
@@ -0,0 +1,190 @@
+{ lib, stdenv
```

```suggestion
{ lib
, stdenv
```
Doing it like you did was just an accident from a treewide change.